mxcube3/ui/components/SampleQueue/HistoryTree.js

```diff
@@ -20,14 +20,17 @@ export default class HistoryTree extends React.Component {
                     <hr className="queue-divider" />
                 </div>
                 <div className={bodyClass}>
-                    {this.props.list.map((id, i) => {
-                      let sampleData = this.props.sampleInformation[this.props.lookup[id]];
+                    {this.props.list.map((sampleId, i) => {
+                      let sampleData = this.props.sampleInformation[sampleId];
                       return (
                         <HistorySampleItem
                           data={sampleData}
-                          show={this.props.collapsedSamples[id]}
+                          show={this.props.displayData[sampleData.sampleID].collapsed}
```

Comment:
You are right it should be sampleData.id, infact I'm using just sampleId here. That's a bit strange I might have missed that in a commit somehow.